### DISCUSSION TITLE
Can I see my framework's benchmark numbers before merging the PR?

### DISCUSSION THREAD

**Discussion by tusharmath** [TechEmpower/FrameworkBenchmarks#6455] [Q&A]
We have been working on adding ZIO Http to the benchmark suite. Can we see the results of the benchmarks before the PR is merged?
The link posted in the build looks like — 

```
Results are saved in /FrameworkBenchmarks/results/20210316121045
```

**Comment by NateBrady23** [comment]
You can visualize the results of local testing by dropping them here: https://www.techempower.com/benchmarks/#section=test

**Reply by tusharmath** [reply]
Those results might differ since it is running in a different environment.
